crates/glaredb/src/highlighter.rs

```diff
@@ -117,7 +117,19 @@ fn colorize_sql(query: &str, st: &mut StyledText, is_hint: bool) -> std::io::Res
                 | Keyword::DROP
                 | Keyword::CREDENTIALS
                 | Keyword::OPTIONS
-                | Keyword::VALUES => {
+                | Keyword::VALUES
+                | Keyword::CASE
+                | Keyword::WHEN
+                | Keyword::THEN
+                | Keyword::IF
+                | Keyword::ELSE
+                | Keyword::END
+                | Keyword::UPDATE
+                | Keyword::SET
+                | Keyword::DELETE
+                | Keyword::VIEW
+                | Keyword::EXCEPT
+                | Keyword::EXCLUDE => {
```

Comment:
I keep forgetting to open up a PR to add `explain`. 
```suggestion
                | Keyword::EXPLAIN
                | Keyword::EXCLUDE => {
```